src/ensembl/src/shared/components/external-link/ExternalLink.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { ReactComponent as LinkIcon } from 'static/img/shared/icon-xlink.svg';
+
+import styles from './ExternalLink.scss';
+
+export type ExternalLinkProps = {
+  label?: string;
+  linkUrl: string;
+  linkText: string;
+  classNames?: {
+    containerClass?: string;
+    labelClass?: string;
+    iconClass?: string;
+    linkClass?: string;
```

Comment:
the word "class" can easily be omitted in all four fields (compare with the names you use for class names on the `styles` object — they do not have the word "class" in them)

Reply:
That's done